apps/docs/content/components/OrderedList.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Ordered List
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/branch/NzBMPIfPrFU9I87d3k7KVq/Primer-Brand?node-id=4045%3A17560&t=W8B9ArDngbzJCC2m-0'
+source: https://github.com/primer/brand/tree/main/src/List/OrderedList/OrderedList.tsx
+storybook: '/brand/storybook/?path=/story/components-orderedlist--default'
+description: Use the OrderedList component to render a list of ordered items.
+---
+
```

Comment:
FYI, there's an issue with the import in docs. Previews aren't working.

![Screenshot 2023-01-10 at 12 11 27](https://user-images.githubusercontent.com/13340707/211550750-3e599d8f-e994-43ea-81c5-9e326409491c.png)

Reply:
In next push